### DISCUSSION TITLE
Mzmine 4.2 in macOS

### DISCUSSION THREAD

**Discussion by adrianmunoz18** [mzmine/mzmine#2017] [Q&A] [answered]
My operating system: macOS Sonoma 14.5
My processor: Dual Core Intel
Hello, is the new version of MZmine working on macOS?
I have problems installing the app.
I followed the directions for installing on Mac and still a message of error is appearing.

**Comment by ansgarkorf** [comment, marked as answer]
We no longer ship for intel based macs due to github limitations.

**Reply by ktarazona** [reply]
Hello, I'm also having problems installing the latest version on Apple silicon mac. 
Operating system: macOS Sonoma 14.6.1
Processor: Apple M2 Pro

Version 4.1.0 works, but I get an error message with 4.2.0.

**Reply by ansgarkorf** [reply]
Can you share the error message?
